charts/benchmark/README.md

```diff
@@ -0,0 +1,137 @@
+# wasmCloud Benchmarking Chart
+
+This chart is primarily used for benchmarking wasmCloud deployments within a Kubernetes cluster. It
+can be used to test something outside of the cluster as well, but this is not a tested use case.
+This chart has a self contained stack of Grafana, Loki, Tempo, OTEL collectors, and Prometheus as
+well as installation for the [K6 Operator](https://github.com/grafana/k6-operator) for running the
+actual benchmark. This setup is meant to be temporary as many users are likely to have their own
+Grafana or OTEL stacks. This setup is purely for capturing data for the test. By default, this is a
+"push button" chart that will install the stack and run the benchmark for you. It can then be
+uninstalled so as to not take up resources.
+
+## Basic Usage
```

Comment:
I think a short tidbit about how to deploy the most basic wasmCloud host in a way that's accessible to this test would be useful here. I actually expected this chart to include the wasmCloud deployment too, e.g. the individual `helm upgrade --install` to deploy wasmCloud and the benchmark test.

For dummies like me that just wanna try it out, would be great to have the oneliner/twoliner to deploy the wasmCloud chart too. Thoughts?

Reply:
So my logic here was benchmarking a system you don't even have installed doesn't have a lot of of value IMO. I do think it would be useful though to add something to the README pointing to the different ways to install

Reply:
Added some more stuff to the README about this. Resolve if it looks good!